jobs/build_tag_monger_triggers.groovy

```diff
@@ -1,25 +0,0 @@
-import util.Common
-Common.makeFolders(this)
-
-def j = job('sqre/infra/build-tag-monger-triggers') {
-  scm {
-    git {
-      remote {
-        github('lsst-sqre/tag-monger')
```

Comment:
Looking at what this repo does, I think something like it may be necessary.  We can't keep all eups package artifacts forever.  But we do want to keep official releases, so a simple time-based lifecycle policy won't work.

That doesn't mean we need to keep this job, but we need to plan to replace this code somehow.